exercise/express-post-json/index.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable indent */
+/* eslint-disable eol-last */
+/* eslint-disable semi */
+/* eslint-disable no-console */
+/* eslint-disable quotes */
+/* eslint-disable no-unused-vars */
+
+const express = require('express');
+const app = express();
+const grades = [];
+let nextId = 1;
+const jsonMiddleware = express.json();
+app.use(jsonMiddleware);
+app.get('/api/grades', (req, res, next) => {
+    res.json(grades);
+  })
+
+  app.post('/api/grades', (req, res, next) => {
+    const body = req.body;
+    body.id = nextId++;
+    res.json(grades);
```

Comment:
The server should respond with only the newly added grade and not the entire list.